src/Compilers/CSharp/Portable/Syntax/SyntaxKind.cs

```diff
@@ -143,8 +143,8 @@ public enum SyntaxKind : ushort
         PercentEqualsToken = 8283,
         /// <summary>Represents <c>??=</c> token.</summary>
         QuestionQuestionEqualsToken = 8284,
-        /// <summary>Represents <c>!!</c> token.</summary>
-        ExclamationExclamationToken = 8285,
+        // Don't use 8285. It corresponds to the deleted kind ExclamationExclamationToken which was previously shipped.
```

Comment:
>shipped [](http://example.com/codeflow?start=111&length=7)

nit: It was not exactly "shipped" since it remained in preview. Consider "Don't use 8285. Was ExclamationExclamationToken before the feature was removed from C# 11"